cross/he853/Makefile

```diff
@@ -0,0 +1,23 @@
+PKG_NAME = he853-cli
+PKG_VERS = 1.0.3
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
```

Comment:
Add `PKG_DIST_NAME = PKG_NAME-$(PKG_VERS).$(PKG_EXT)` here, so the downloaded file has a unique name.
Regenerate the digests file, so it has the correct name in it.